src/components/MultiHopTrade/components/TradeAmountInput.tsx

```diff
@@ -219,19 +213,29 @@ export const TradeAmountInput: React.FC<TradeAmountInputProps> = memo(
         <Flex
           direction='row'
           gap={2}
-          py={2}
-          px={4}
+          pt={4}
+          pb={2}
+          px={6}
           justifyContent='space-between'
           alignItems='center'
         >
-          {balance && (
-            <Balance
-              cryptoBalance={balance}
-              fiatBalance={fiatBalance ?? ''}
-              symbol={assetSymbol}
-              isFiat={isFiat}
-              label={translate('common.balance')}
-            />
+          {showFiatAmount && (
+            <Button
+              onClick={toggleIsFiat}
+              size='sm'
+              disabled={showFiatSkeleton}
+              fontWeight='medium'
+              variant='link'
+              color='text.subtle'
+            >
+              <Skeleton isLoaded={!showFiatSkeleton}>
+                {isFiat ? (
+                  <Amount.Crypto value={cryptoAmount ?? ''} symbol={assetSymbol} />
+                ) : (
+                  <Amount.Fiat value={fiatAmount ?? ''} prefix='≈' />
+                )}
```

Comment:
might be worth putting this in a useMemo as it creates a new element every render which matters here since this component is high up the tree